jelatyna-backend/src/main/java/pl/confitura/jelatyna/agenda/InlineAgenda.java

```diff
@@ -3,7 +3,7 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.data.rest.core.config.Projection;
 import pl.confitura.jelatyna.presentation.Presentation;
-import pl.confitura.jelatyna.user.PublicUser;
+import pl.confitura.jelatyna.user.dto.PublicUser;
```

Comment:
Zastanawiam sie, czy DTO nie powinien mieć jakiegoś przyrostka zeby był łatwo identyfikowalny... albo `DTO` albo `Request`/`Response`.

Co myślisz?

Reply:
tych dto bedziemy uzywac czesciej niz klasy-encji wiec chcialem sprawdzic jak to sie bedzie zachowywac, ale jednak chyba zrobie z dto - jest czytelniej

Reply:
Masz rację, w takim razie może model nazwać `UserModel`, a obiekty DTO prostymi nazwami (bez suffixów)